app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+from app.errors import (

```

Comment:
should be a blank line between different types of imports. Also it is a good practice to imports alphabetically